packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -31,7 +32,20 @@ const Radius _kToolbarBorderRadius = Radius.circular(8);
 
 // Colors extracted from https://developer.apple.com/design/resources/.
 // TODO(LongCatIsLooong): https://github.com/flutter/flutter/issues/41507.
-const Color _kToolbarDividerColor = Color(0xFF808080);
+const Color _kToolbarDividerColorDark = Color(0xFF808080);
```

Comment:
use CupertinoDynamicColor instead?

Reply:
Yes thank you, I forgot that existed but it's so much more convenient 👍